sdc/crypto/test/test_jwt_helper.py

```diff
@@ -1,12 +1,14 @@
 import base64
 import os
 from unittest import TestCase
+import json
 
 from sdc.crypto.exceptions import InvalidTokenException
 from sdc.crypto.jwt_helper import JWTHelper
 from sdc.crypto.key_store import KeyStore
 from sdc.crypto.test import TEST_DO_NOT_USE_UPSTREAM_PUBLIC_PEM, TEST_DO_NOT_USE_SR_PRIVATE_PEM, \
-    TEST_DO_NOT_USE_UPSTREAM_PRIVATE_KEY, TEST_DO_NOT_USE_SR_PUBLIC_KEY
+    TEST_DO_NOT_USE_UPSTREAM_PRIVATE_KEY, TEST_DO_NOT_USE_SR_PUBLIC_KEY, TEST_DO_NOT_USE_EQ_PRIVATE_KEY, \
```

Comment:
Could use parentheses instead of line breaks for imports.

Reply:
could do, but its what is used throughout this repo so for consistency just keep it same